pallets/funding/src/tests/3_auction.rs

```diff
@@ -1960,3 +1662,356 @@ mod bid_extrinsic {
+			let bounded_name = bounded_name();
+			let bounded_symbol = bounded_symbol();
+			let metadata_hash = ipfs_hash();
+			let normalized_price = PriceOf::<TestRuntime>::from_float(10.0);
+			let decimal_aware_price = PriceProviderOf::<TestRuntime>::calculate_decimals_aware_price(
+				normalized_price,
+				USD_DECIMALS,
+				CT_DECIMALS,
+			)
+			.unwrap();
+			let project_metadata = ProjectMetadata {
+				token_information: CurrencyMetadata {
+					name: bounded_name,
+					symbol: bounded_symbol,
+					decimals: CT_DECIMALS,
+				},
+				mainnet_token_max_supply: 8_000_000 * CT_UNIT,
+				total_allocation_size: 100_000 * CT_UNIT,
+				auction_round_allocation_percentage: Percent::from_percent(50u8),
+				minimum_price: decimal_aware_price,
+				bidding_ticket_sizes: BiddingTicketSizes {
+					professional: TicketSize::new(5000 * USD_UNIT, None),
+					institutional: TicketSize::new(5000 * USD_UNIT, None),
+					phantom: Default::default(),
+				},
+				contributing_ticket_sizes: ContributingTicketSizes {
+					retail: TicketSize::new(USD_UNIT, None),
+					professional: TicketSize::new(USD_UNIT, None),
+					institutional: TicketSize::new(USD_UNIT, None),
+					phantom: Default::default(),
+				},
+				participation_currencies: vec![AcceptedFundingAsset::USDT].try_into().unwrap(),
+				funding_destination_account: ISSUER_1,
+				policy_ipfs_cid: Some(metadata_hash),
+			};
+
+			// overfund with plmc
+			let plmc_fundings = accounts
+				.iter()
+				.map(|acc| UserToPLMCBalance { account: acc.clone(), plmc_amount: PLMC * 1_000_000 })
+				.collect_vec();
+			let usdt_fundings = accounts
+				.iter()
+				.map(|acc| UserToFundingAsset {
+					account: acc.clone(),
+					asset_amount: USD_UNIT * 1_000_000,
+					asset_id: AcceptedFundingAsset::USDT.id(),
+				})
+				.collect_vec();
+			inst.mint_plmc_to(plmc_fundings);
+			inst.mint_funding_asset_to(usdt_fundings);
+
+			let project_id =
+				inst.create_auctioning_project(project_metadata.clone(), ISSUER_1, None, default_evaluations());
+
+			let bids = vec![
+				(ADAM, 10_000 * CT_UNIT).into(),
+				(TOM, 20_000 * CT_UNIT).into(),
+				(SOFIA, 20_000 * CT_UNIT).into(),
+				(FRED, 10_000 * CT_UNIT).into(),
+				(ANNA, 5_000 * CT_UNIT).into(),
+				(DAMIAN, 5_000 * CT_UNIT).into(),
+			];
+
+			inst.bid_for_users(project_id, bids).unwrap();
+
+			assert!(matches!(inst.go_to_next_state(project_id), ProjectStatus::CommunityRound(..)));
+
+			let project_details = inst.get_project_details(project_id);
+			dbg!(&project_details);
+			let token_price = inst.get_project_details(project_id).weighted_average_price.unwrap();
+			let normalized_wap =
+				PriceProviderOf::<TestRuntime>::convert_back_to_normal_price(token_price, USD_DECIMALS, CT_DECIMALS)
+					.unwrap();
+			let desired_price = PriceOf::<TestRuntime>::from_float(11.1818f64);
+
+			assert_close_enough!(
+				normalized_wap.saturating_mul_int(CT_UNIT),
+				desired_price.saturating_mul_int(CT_UNIT),
+				Perquintill::from_float(0.99)
```

Comment:
Maybe update to 0.999 (0.1% mismatch instead of 1%). As 1% is quite a big impact on price if it's incorrect.